File: Web/src/app/models/usuario.ts
export class Usuario {
  id: number;
  nome: string;
  login: string;
  password: string;
  setor: UsuarioSetor;
}

export class Autenticacao {
  authenticated: boolean;
  created: Date;
  dateExpiration: Date;
  token: string;
  usuario: Usuario;
  Message: string;
}

export enum UsuarioSetor {
  Garcon,
  Cozinha
}

File: Api/FavoDeMel.Domain/Usuarios/Usuario.cs
﻿using FavoDeMel.Domain.Common;

namespace FavoDeMel.Domain.Usuarios
{
    public class Usuario : Entity<int>
    {
        public string Nome { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
        public UsuarioSetor Setor { get; set; }
    }
}
